pkgs/applications/editors/vim/plugins/non-generated/blink-cmp/default.nix

```
@@ -41,10 +41,16 @@ vimUtils.buildVimPlugin {
     ''
       mkdir -p target/release
       ln -s ${blink-fuzzy-lib}/lib/libblink_cmp_fuzzy${ext} target/release/libblink_cmp_fuzzy${ext}
+      echo -n "nix" > target/release/version
     '';
 
+  # TODO: Remove this patch when updating to next version
   patches = [
-    (replaceVars ./force-version.patch { inherit (src) tag; })
+    (fetchpatch {
+      name = "blink-add-bypass-for-nix.patch";
+      url = "https://github.com/Saghen/blink.cmp/commit/6c83ef1ae34abd7ef9a32bfcd9595ac77b61037c.patch";
```

For better reproducibility 
```suggestion
      url = "https://github.com/Saghen/blink.cmp/commit/6c83ef1ae34abd7ef9a32bfcd9595ac77b61037c.patch?full_index=1";
```